changelog.markdown

```diff
@@ -3,6 +3,39 @@
 
 # Nimble changelog
 
+## 0.11.6 - TBD
+
+This is a minor release containing just X commits. This release brings mostly
+bug fixes with some minor improvements:
+
+- `nimble dump` now provides `--json` output.
+- Calls to the Nim compiler now display `--hints:off` output by default.
+  The `--verbose` flag will print the full Nim command as well as regular
+  compiler output.
+- Custom tasks can now be passed compiler flags as well as run flags when run
+  as `nimble <compflags> task <runflags>`. This includes the custom `test`
+  task if defined. Compile flags can be used to set `--define:xxx` values that
+  can be checked with `when defined(xxx)` and other compiler flags that are
+  applicable in nimscript mode. Run flags can be accessed per usual from
+  `commandLineParams: seq[string]`.
```

Comment:
how are these compile flags and run flags used? Are they passed implicitly to any ``exec "nim .."`` calls done in the task? You should explain that explicitly.

Reply:
Not to the exec calls but to the `nim e` call that processes the task. I'll clarify.

User will have to use the run flags to manually pass stuff to the exec calls if they want.